sdk/java/dagger-java-sdk/src/main/java/io/dagger/module/info/StringOptionInfo.java

```diff
@@ -0,0 +1,3 @@
+package io.dagger.module.info;
+
+public record StringOptionInfo(boolean isSet, String value) {}
```

Comment:
why not using `Optional<String>` instead?

Reply:
Well... I don't know :-D
I'll give a try 👍

Reply:
Updated with `Optional` 👍